blueprint_agreement.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
   spec.add_runtime_dependency "activesupport", ">= 4.2", "< 5.1"
+  spec.add_runtime_dependency "minitest", "~>5.9"
```

Comment:
@pedrofernandezm  change this for development dependency and see if everything is working